ManageIQ/manageiq
# [EUWE] Add Gnocchi service for Openstack metrics to Euwe

Gnocchi provides C&U functionality since it was moved form Ceilometer API. We need to add a support for the service. This implementation should be able handle both Ceilometer and Gnocchi based metric capture.

Links
----------------

* https://bugzilla.redhat.com/show_bug.cgi?id=1387310
* Required Fog/Openstack change https://github.com/fog/fog-openstack/pull/224
* upstream PR https://github.com/ManageIQ/manageiq/pull/12396

Steps for Testing/QA
-------------------------------

* setup OSP10 and add it as Cloud provider
* enable metrics collection
* provision a VM
* invoke metrics capture or wait until it gets captured
* open Monitoring - Utilization on the VM detail page and you should see charts with meaningful data
